core/theme.js

```diff
@@ -57,27 +57,44 @@ Blockly.Theme = function(name, blockStyles, categoryStyles,
    * @private
    */
   this.componentStyles_ = opt_componentStyles || Object.create(null);
+
+  /**
+   * The font style.
+   * @type {?Blockly.Theme.FontStyle}
+   * @package
+   */
+  this.fontStyle = null;
```

Comment:
Should we add a getter for this? So someone could change just a single value on a theme instead of having to change the entire fontStyle?

Reply:
I can make it public instead of package, and then users can access it directly.

Reply:
But then we wouldn't know about changes to it -> wouldn't know to update the constant provider.

Reply:
So right now the way we know the theme has changed at all is by the user calling ``workspace.setTheme(workspace.getTheme())``. Trigger a re-fresh.

We could call a refresh when the theme.setX is called, trouble with that is what if you're making multiple changes, ie: setX, setY ..., we'd be calling a refresh on every set.

Reply:
Ah right, we're not trying to be magical about detecting these changes (and don't need to be).